### PR TITLE
Worker portfolio back end integration

### PR DESCRIPTION
**Pull request by Macbull**
Right now, models does not relate a module to any skill or skill level, so that is left as constant.

### CODE REVIEW

**Comment by dmorina** [comment]
They do, via module qualifications

**Comment by Macbull** [comment]
qualification model is connected to module, but skill model is only connected to Worker via workerskills

**Comment by dmorina** [comment]
Yeah and a qualification attribute can be a skill, we will cover this later don't worry about it

File: staticfiles/js/worker/services/worker.service.js [hunk 1]
```diff
@@ -29,16 +29,16 @@
...
-    }   
+
+    }
 
     function getWorkerTaskPortfolio() {
       return $http({
-        url: 'https://api.myjson.com/bins/q7jc',
+        url: '/api/worker/1/portfolio/',
         method: 'GET'
       });
```
Comment (nistala) [inline staticfiles/js/worker/services/worker.service.js]: use HttpService when you're calling the api

Comment (Macbull) [inline staticfiles/js/worker/services/worker.service.js]: I tried that, but the whole page goes blank, after using that . Getting 304 response code !

Comment (nistala) [inline staticfiles/js/worker/services/worker.service.js]: you're not sending auth headers, so this won't work once auth is enabled.
304 means not modified, so something is wrong with the implementation

Comment (Macbull) [inline staticfiles/js/worker/services/worker.service.js]: I don't know what's going wrong ? HttpService doesn't seems to work with detail routes .

Comment (dmorina) [inline staticfiles/js/worker/services/worker.service.js]: httpservice should work just fine, the backend is not dependent on that

Comment (Macbull) [inline staticfiles/js/worker/services/worker.service.js]: The problem is solved , if I put 'return' before $http(settings).success(function (data,config) {
        deferred.resolve(arguments); in http.service.js

Comment (Macbull) [inline staticfiles/js/worker/services/worker.service.js]: the response code 304 was for static files, not for api request, sorry for confusion. But the problem still exist, I think HttpService is not returning response.

**Comment by dmorina** [comment]
resolve merge conflicts and split this into two PRs, please do only one thing in one PR, it makes it easier to check and merge and that's how it's supposed to be :)

**Pull request closed without merging**